### PR TITLE
Fix returning data and error for multiple resolvers

### PR DESCRIPTION
**Pull request by DifferentialOrange**
Rework of #19 . Additional test was reworked so both resolvers return error. Added description to commit message, removed #18 commit from PR.

### CODE REVIEW

**Review by olegrok — APPROVED** [review]

File: graphql/execute.lua [hunk 1]
```diff
@@ -313,7 +313,6 @@ end
```
Comment (Totktonada) [inline graphql/execute.lua:313]: > This PR fixes the logic to collect all errors from all resolvers
> into a single map that can be returned to requester.

Nit: the word 'map' it was a bit confusing for me in this context (hower the test shows that it is actually an array).

**Review by Totktonada — APPROVED** [review]
LGTM.

File: graphql/execute.lua (hunk 1 above)
Comment (DifferentialOrange) [reply graphql/execute.lua:313]: Fixed

**Comment by DifferentialOrange** [comment]
Resolved all minor comments. PR is ready to merge

**Pull request merged**